README.md

```diff
@@ -0,0 +1,30 @@
+<h3 align="center">vacay store</h3>
```

Comment:
this is a markdown file...so you can use markdown syntax instead of html

for example, use `#` for h1 and `##` for h2 and so on...

`# This is a H1 heading`

Reply:
Thanks for telling that. I wanted to center it therefore I have used html. I saw some popular github repos who have used this.